### ISSUE TITLE
If set to 'Reverse Fill' bar textures are moving to the right.

### ISSUE THREAD

**Issue by vanIbeX** [tukui-org/ElvUI#1180]
### Troubleshooting Steps

If you set the bar option 'Reverse Fill' for UnitFrames the bar texture is still left bound. As a result the texture seems to be moving to the right.

This isn't noticable with the default configuration because of the default textures but the behavior should be the same. However, it becomes apparent if you use ElvUI Gloss.

![image](https://github.com/tukui-org/ElvUI/assets/26798341/5f1ebc33-49f5-476d-af99-6d00c24e6e53)

### Describe the expected behavior and what actually happened?

If the setting 'Reverse Fill' is active, I would expect the bar texture to also reverse fill/be mirrored.

### Reproducing the issue.

Set a bar to 'Reverse FIll' and use a patterned bar texture (like ElvUI Gloss). If the value of the bar changes, the texture will be moved to the right.

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by 1Botanica1** [comment]
Please do the troubleshooting as asked when creating the ticket and provide the information requested. 

Type in game `/edebug on`

Test - do you have the issue?

If so, type in chat in game `/estatus` and upload a FULL UI screenshot (no cropping) showing both the error and the status panel.

**Comment by vanIbeX** [comment]
I did the test. The issue still persists. Here is the full uncropped picture of the issue with a twink.

![Unbenannt](https://github.com/tukui-org/ElvUI/assets/26798341/7b0d4a12-2057-404d-9ee9-2c81ebed0cf4)

**Comment by 1Botanica1** [comment]
You haven’t followed all the steps above.  You have other addons enabled. 

The first step is important

1. Type in game `/edebug on`

2. Test - do you have the issue?

3. If so, type in chat in game `/estatus` and upload a FULL UI screenshot (no cropping) showing both the error and the status panel.

**Comment by vanIbeX** [comment]
Yep, sorry. Now it should be correct. As the debug bars get darker to the left you can see the same behaviour.

![Unbenannt](https://github.com/tukui-org/ElvUI/assets/26798341/b8f7f579-a2ca-435a-b6d9-0c7646c1e7a8)

**Comment by vanIbeX** [comment]
Comment in hope to maybe get a statement regarding this issue. Is this something that will get any attention?
